src/pages/Main/PostList/PostList.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import { setFlex, theme } from '../../../styles/theme';
+function Post(post) {
```

Comment:
```suggestion

function Post(props) {
```
- 구분되는 요소 한칸 띄워주세요
- setFlex, theme 등은 삭제해주시고 앞서 남겨드린 리뷰 확인해주세요
- 함수 컴포넌트의 인자는 컴포넌트의 props 자리기 때문에 일반적으로 props라고 적어주면 됩니다

Reply:
넵!